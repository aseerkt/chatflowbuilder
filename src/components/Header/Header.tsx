export default function Header({ children }: { children: React.ReactNode }) {
  return (
    <header className='flex h-[90px] items-center justify-between border-b bg-orange-50 px-6'>
      <h1 className='text-2xl font-bold'>Chat Flow Builder</h1>
      {children}
    </header>
  )
}
