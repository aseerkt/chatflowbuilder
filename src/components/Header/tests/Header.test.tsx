import { render } from '@testing-library/react'
import Header from '../Header'

describe('<Header /> tests', () => {
  it('should match snapshot', () => {
    const { baseElement } = render(<Header>Hello World</Header>)
    expect(baseElement).toMatchSnapshot()
  })
})
