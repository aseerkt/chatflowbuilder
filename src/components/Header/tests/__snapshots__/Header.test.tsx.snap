// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Header /> tests > should match snapshot 1`] = `
<body>
  <div>
    <header
      class="flex h-[90px] items-center justify-between border-b bg-orange-50 px-6"
    >
      <h1
        class="text-2xl font-bold"
      >
        Chat Flow Builder
      </h1>
      Hello World
    </header>
  </div>
</body>
`;
